build/release.mk

```diff
@@ -0,0 +1,42 @@
+BUILD_DIR    := ./bin/
+GORELEASER_VERSION := v0.143.0
```

Comment:
We can try with latest release of .goreleaser [v0.155.0](https://github.com/goreleaser/goreleaser/releases/tag/v0.155.0)